### ISSUE TITLE
Should webmention ping data support extensibility via something like LD namespaces?

### ISSUE THREAD

**Issue by gobengo** [w3c-social/webmention#2]
Started on #1, but that issue is specifically to introduce a new parameter in the currently specced `application/x-www-form-urlencoded` request body content-type.

> Webmention has been implemented in linked data in the form of "Semantic Pingback" http://www.w3.org/wiki/Pingback#Semantic_Pingback

Semantic Pingback is worth being familiar with in the webmention discussion. However, that document does not use normative vocabulary, and thus I don't think it is fit as-is.

@melvincarvalho :
> I'd suggest to use namespaces for source, target etc. from the pingback vocab if wishing to implement this in the linked data world. Alternatively putting something under w3.org/ns could work.

Which may or may not be a good idea. Regardless, I've never seen LD namespaces in a `application/x-www-form-urlencoded` string. So perhaps it is a good idea to standardize webmention pings in other Content-Types for use cases where more than just a 'source' and 'target' are known.

Iff webmention does standardize LD-friendly Content-Type ping bodies, I agree that it's a good idea to re-use (or explicitly equivalent) the pingback namespace (esp as it already defines `source` and `target` as in current webmention draft).

It seems to me that all the debates this morning about 'namespaces' really imply a request for further discussion on Content-Types.

**Comment by melvincarvalho** [comment]
+!

**Comment by dissolve** [comment]
I'd support creating some extensions to webmention (look at indiewebcamp.com/vouch for one such extension).
Remember though that any extension really should be limited to things needed for transfer not data itself, which is a pretty limited set of things (auth, spam prevention, fetch format, etc).   That's why I think namespacing might be overkill

**Comment by tantek** [comment]
Without starting with a specific documented real-world use-case that requires something above & beyond the existing spec, we should summarily close this issue. Plumbing for plumbing's only bloats specs, makes more work for implementers without benefit to their implementations or their users.

**Comment by gobengo** [comment]
Given that there is a precedent (vouch) of defining extensions for form-urlencoded fields other than `source` and `target`.
and given that we've identified that extended form-urlencoded fields could indicate LD context, but that is a different problem outside of scope of socialwg,
and noting there is not currently a list of use cases driving this extensibility story,
I agree with Tanktek for now.

I opened the issue to isolate the discussion from #1. But don't see it as an issue with the current, nicely small, webmention spec.

So I'm closing because I'm -1 unless relevant use cases require it from the Core (as opposed to extension). But now this topic is here and discussion can continue with context.
